GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -259,12 +258,30 @@ void Renderer::frustum_cull(Scene* scene,
 
         // Only push those scene objects that are inside of the frustum
         if (!is_inside) {
-            (*it)->set_in_frustum(false);
+            scene_object->set_in_frustum(false);
+            continue;
+        }
+
```

Comment:
How about adding another api here, e.g. scene_object->isLODUsed()? If false, we can skip calculating steps of camera distance (the default is in-range when lod is not in use)?
It can be a minor optimization considering setLODRange() is currently set only from app but frustum_cull() is in the framework.